### ISSUE TITLE
Archive failed

### ISSUE THREAD

**Issue by aabanaag** [SDWebImage/SDWebImage#3169]
### New Issue Checklist

* [X] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [X] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [X] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | iOS
 Platform Version        | 
 SDWebImage Version      | 5.10.3
 Integration Method      | cocoapods
 Xcode Version           | Xcode 12
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

When building via Bitrise, I get this error despite updating the pods already.
`The following build commands failed:
	CompileC /Users/vagrant/Library/Developer/Xcode/DerivedData/MyTotalHealth-amgsqhnmugghbzexqobmycxuzdzp/Build/Intermediates.noindex/ArchiveIntermediates/Test/IntermediateBuildFilesPath/Pods.build/Test-iphoneos/SDWebImage.build/Objects-normal/armv7/SDWeakProxy.o /Users/vagrant/git/Pods/SDWebImage/SDWebImage/Private/SDWeakProxy.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)`

`CompileC /Users/vagrant/Library/Developer/Xcode/DerivedData/MyTotalHealth-amgsqhnmugghbzexqobmycxuzdzp/Build/Intermediates.noindex/ArchiveIntermediates/Test/IntermediateBuildFilesPath/Pods.build/Test-iphoneos/SDWebImage.build/Objects-normal/armv7/SDImageCachesManagerOperation.o /Users/vagrant/git/Pods/SDWebImage/SDWebImage/Private/SDImageCachesManagerOperation.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'SDWebImage' from project 'Pods')`

Error from bitrise
`❌  /Users/vagrant/git/Pods/SDWebImage/SDWebImage/Private/SDWeakProxy.h:10:9: 'SDWebImageCompat.h' file not found
#import "SDWebImageCompat.h"
        ^~~~~~~~~~~~~~~~~~~~`
